lib/utils/util.js

```diff
@@ -229,3 +230,34 @@ util.createBatch = function(limit, domainOutputMap) {
 
   return {validDomains: validDomains, rejectedDomains: rejectedDomains};
 };
+
+/**
+ * Transform BatchBids in to
+ * Expected format
+ * @param {Object} txJSON
+ * @param {Object} mtx
+ * @returns {Array<Bid>} processedBids
+ */
+
+util.postProcessBatchBids = function(txJSON, mtx) {
+  const processedBids = [];
+  //
+  const mtxOutputs = mtx.outputs;
+  const txOutputs = txJSON.outputs;
+  //
+  for (let i=0; i<txOutputs.length; i++) {
```

Comment:
Can you send me your global eslint config file, my editor really not showing there errors, it would help me to catch these type of things from the very start ...

Reply:
Update: https://github.com/handshake-org/hsd/issues/539

I'm going to stop commenting on these style errors, ideally try to maintain the style that the rest of the code follows, but style is never something that should come up in a PR review in my opinion. The tooling should make it impossible to mess up the style so that programmers don't need to debate it. I want to scope in some time to codify hsd's implicit style rules some time after this batch bid work.